modules/datastore/src/Service/PostImport.php

```diff
@@ -219,13 +207,8 @@ protected function createPostImportResult($status, $message, DataResource $resou
    *   The resource identifier of the distribution.
    */
   public function drop($resourceIdentifier): bool {
-    try {
-      $this->datastoreService->drop($resourceIdentifier, NULL, FALSE);
-      return TRUE;
-    }
-    catch (\Exception $e) {
-      throw $e;
-    }
+    $this->datastoreService->drop($resourceIdentifier, NULL, FALSE);
+    return TRUE;
```

Comment:
Boggle. How did we end up with a no-op try/catch in the first place?

Reply:
lol no idea